client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -119,21 +154,70 @@ function handleError(err: any) {
         jobError.value = err.data;
     }
 }
+
+function onAttribute(data: Record<string, any>) {
+    for (const key in data) {
+        const index = attributesInputs.value?.findIndex((input) => input.name === key);
+        if (index !== -1 && attributesInputs.value[index]) {
+            attributesInputs.value[index]!.value = data[key];
+        }
+    }
+}
+
+async function saveAttrs() {
+    if (collection.value && attributesInputs.value) {
+        const updatedAttrs = attributesInputs.value.reduce((acc, input) => {
+            acc[input.name] = input.value;
+            return acc;
+        }, {} as Record<string, any>);
+        try {
+            await updateContentFields(collection.value, updatedAttrs);
+
+            successMessage.value = "Attributes updated successfully.";
+        } catch (err) {
+            errorMessage.value = errorMessageAsString(err, "Unable to update attributes.");
+        }
+    }
+}
 </script>
 
 <template>
     <div aria-labelledby="collection-edit-view-heading">
-        <h1 id="collection-edit-view-heading" class="h-lg">{{ localize("Edit Collection Attributes") }}</h1>
+        <Heading id="dataset-attributes-heading" h1 separator inline size="xl">
+            {{ localize("Edit Collection Attributes") }}
+        </Heading>
 
-        <BAlert show variant="info" dismissible>
+        <BAlert v-if="infoMessage" show variant="info" dismissible>
             {{ localize(infoMessage) }}
         </BAlert>
 
         <BAlert v-if="jobError" show variant="danger" dismissible>
             {{ localize(errorMessage) }}
         </BAlert>
 
-        <BTabs content-class="mt-3">
+        <BAlert v-if="successMessage" show variant="success" dismissible>
+            {{ localize(successMessage) }}
+        </BAlert>
+        <BTabs class="mt-3">
+            <BTab title-link-class="collection-edit-attributes-nav" @click="updateInfoMessage('')">
+                <template v-slot:title>
+                    <FontAwesomeIcon :icon="faBars" class="mr-1" />
+                    {{ localize("Attributes") }}
+                </template>
+
+                <FormDisplay
```

Comment:
Could we use explicit input components here instead e.g. `FormInput`, or are the inputs here dynamic / unknown to the client?

Reply:
Oh I think we could easily use explicit components. I was just following what we had in `DatasetAttributes`, but we get those form inputs from `/dataset/get_edit?dataset_id=${datasetId}` (which will be deprecated?)

Having the `FormInputs` also allows us to add any more fields (other than name) later...?

Reply:
I think mirroring DatasetAttributes makes sense here for now, this looks good to me.